### PR TITLE
A word on the DB model; closes #6

### LINKED ISSUE #6: Backport db model doc
https://forge.glpi-project.org/projects/glpi/wiki/DBmodel

### PR DESCRIPTION
**Pull request by trasher**
See http://glpi-developer-documentation.readthedocs.io/en/feature-dbmodel/devapi/dbmodel.html

### CODE REVIEW

File: source/devapi/dbmodel.rst [hunk 1]
```diff
@@ -0,0 +1,97 @@
+Database model
+--------------
+
+Current GLPI database contains more than 250 tables; the goal of the current documentation is to help you to understand the logic of the project, but not to detail each table and possibility.
```
Comment (orthagh) [inline source/devapi/dbmodel.rst]: ", but" is superflu, imo

File: source/devapi/dbmodel.rst [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+Naming conventions
+^^^^^^^^^^^^^^^^^^
+
+All tables and fields names are lower case and follows the same logic. If you do not respect that; GLPI will fail to find relevant informations.
+
+Tables
+++++++
+
+Tables names are linked with PHP classes names; they are all prefixed with ``glpi_``, and class name is set to plural. Plugins tables must be prefixed by ``glpi_plugin_``; followed by the plugin name, another dash, and then pluralized class name.
```
Comment (orthagh) [inline source/devapi/dbmodel.rst]: a word on the case maybe ?

File: source/devapi/dbmodel.rst [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+      PRIMARY KEY (`id`),
+      KEY `items_id` (`items_id`),
+      KEY `devicememories_id` (`devicememories_id`),
+      KEY `itemtype` (`itemtype`,`items_id`),
+   ) ENGINE=MyISAM DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;
+
+Again, this is a very simplified example of what already exists in the database, but you got the point ;)
+
+In this example, ``itemtype`` would be ``Computer``, ``Printer`` or ``Phone``; ``items_id`` the ``id`` of the related item.
+
```
Comment (orthagh) [inline source/devapi/dbmodel.rst]: I think the concept of itemtype/items_id should be more important, it's a very common way.

File: source/devapi/dbmodel.rst (hunk 2 above)
Comment (trasher) [inline source/devapi/dbmodel.rst]: Case is mentioned in "Naming conventions" globally

File: source/devapi/dbmodel.rst (hunk 3 above)
Comment (trasher) [inline source/devapi/dbmodel.rst]: I don't understand what you want to change here

**Pull request merged**